src/test/java/com/google/api/codegen/testdata/ruby/ruby_main_longrunning.baseline

```diff
@@ -83,10 +83,6 @@ module Google
       ALL_SCOPES = [
       ].freeze
 
-      # @param service_path [String]
```

Comment:
Let's deprecate these. We can remove when we declare GA. (See as example: https://github.com/googleapis/toolkit/pull/1387/files#diff-58544da0694fa700a3f27fc817ec47d4R160)